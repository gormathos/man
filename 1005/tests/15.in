-6 -3 -6 -4 7 -5 -4 -3 0 4 -5
-8 -4 1 -10 7 -9 4 4 4 6 10
9 -6 2 -5 10 4 -5 -5 -4 7 6
3 -7 -2 6 10 -3 -1 -8 -9 9 -3
-8 10 6 -3 -6 1 -6 -6 -5 9 0
5 10 5 5 -8 -2 -5 -9 6 6 0
6 5 6 10 1 -2 7 0 7 8 -7
7 -1 -9 7 0 -2 -2 3 2 -2 -9
8 -8 9 7 -3 -9 3 -4 -5 7 -1
