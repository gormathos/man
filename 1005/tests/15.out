9
-6 -6 -4 7 -5 -3 0 4 -5
-8 1 -10 7 -9 4 4 6 10
9 2 -5 10 4 -5 -4 7 6
3 -2 6 10 -3 -8 -9 9 -3
-8 6 -3 -6 1 -6 -5 9 0
5 5 5 -8 -2 -9 6 6 0
6 6 10 1 -2 0 7 8 -7
7 -9 7 0 -2 3 2 -2 -9
8 9 7 -3 -9 -4 -5 7 -1
9077746646
